py_zipkin/encoding.py

```diff
@@ -0,0 +1,41 @@
+import socket
+from collections import namedtuple
+
+Endpoint = namedtuple(
+    'Endpoint',
+    ['service_name', 'ipv4', 'ipv6', 'port'],
+)
+
+
+def create_endpoint(port=0, service_name='unknown', host=None):
+    if host is None:
+        try:
+            host = socket.gethostbyname(socket.gethostname())
```

Comment:
might be better to not set ipv4 vs passing localhost.

Reply:
Are you talking about setting `127.0.0.1` in the except or this `socker.gethostname`? Github shows this comment in an ambiguous position

Reply:
Are you talking about setting 127.0.0.1 in the except or this socker.gethostname? Github shows this comment in an ambiguous position

Reply:
Done